app.rb

```diff
@@ -0,0 +1,220 @@
+require './person'
+require './book'
+require './rental'
+require './teacher'
+require './student'
+
+# rubocop:disable  Metrics/ClassLength
+class App
+  def initialize
+    @rentals = []
+    @books = []
+    @people = []
+  end
+
+  def choices_list
+    choices = ['1 - List all book',
+               '2 - List all People',
+               '3 - Create a person',
+               '4 - Create a book',
+               '5 - Create a rental',
+               '6 - List all rentals for a given person id',
+               '7 - Exit']
+
+    choices.each { |choice| puts choice.to_s }
+  end
+
+  # rubocop:disable Metrics/CyclomaticComplexity
+  def choose(choice)
+    case choice
+    when 1
+      list_books
+    when 2
+      list_people
+    when 3
+      create_person
+    when 4
+      create_book
+    when 5
+      create_rental
+    when 6
+      list_rentals
+    when 7
+      puts
```

Comment:
- [ ] Kindly remove unused extra lines. When I tried to close the app it shows me extra `nill` value. 


<img width="437" alt="Screen Shot 2022-07-29 at 05 37 33" src="https://user-images.githubusercontent.com/69011963/181659537-d0b4161a-3751-4ad9-bc90-c9a285455bf5.png">